newIDE/app/src/MainFrame/Preferences/PreferencesContext.js

```diff
@@ -206,6 +210,7 @@ export type PreferencesValues = {|
   showCommunityExtensions: boolean,
   showGetStartedSection: boolean,
   showEventBasedObjectsEditor: boolean,
+  defaultWorkspace: string,
```

Comment:
What do you think of naming this 
```suggestion
  defaultWorkspaceFolder: string,
```

or even `newProjectsDefaultFolder` - so that it's very clear? 
The name "workspace" is a bit generic/could be misunderstood for something more complex.

Reply:
Let's go for it then 👍